packages/eslint-plugin/docs/rules/prefer-nullish-coalescing.mdx

```diff
@@ -181,8 +181,6 @@ Also, if you would like to ignore all primitives types, you can set `ignorePrimi
 
 {/* insert option description */}
 
-Whether to ignore expressions that coerce a value into a boolean: `Boolean(...)`.
```

Comment:
Option's description is slightly different: `Whether to ignore arguments to the `Boolean` constructor`.